pkg/skaffold/schema/v3/config.go

```diff
@@ -0,0 +1,1775 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v3
+
+import (
+	"encoding/json"
+
+	v1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/util"
+)
+
+// This config version is not yet released, it is SAFE TO MODIFY the structs in this file.
```

Comment:
should this text say "released"?

Reply:
Yes it should instead have the messageL
// !!! WARNING !!! This config version is already released, please DO NOT MODIFY the structs in this file

this is updated now